### DISCUSSION TITLE
Output.pulseaudio to multiple network servers?

### DISCUSSION THREAD

**Discussion by jschmall** [savonet/liquidsoap#2944] [Q&A]
Right now I am outputting to the same pulseaudio server that I'm ingesting from. I'd like to figure out a way to output to a DIFFERENT network server. Any ideas?

Let me know if I need to provide details of my setup.

**Comment by toots** [comment]
There's an interesting response here: https://unix.stackexchange.com/a/605652 that seems to point to zeroconf discover/publish. Looks like one it's setup properly, you should be able to see different sinks for each machine using `pacmd list-sinks`

**Reply by jschmall** [reply]
Interesting. So with that logic, if I have a sink one server 1 that isn’t on server 2, I shouldn’t need to worry about defining that server instance in the pulse client on my liquidsoap server, only needing to define the “device” and let the mDNS (zeroconf) do the test. Ok I’ll go down this rabbit hole later tonight.
